src/Aspire.Dashboard/Components/Pages/TraceDetail.razor.css

```diff
@@ -285,12 +285,11 @@
 }
 
 ::deep.log-tooltip-table td {
-    padding: calc((var(--design-unit) + var(--focus-stroke-width) - var(--stroke-width))* 1px) 0;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    white-space: nowrap;
+    padding: calc((var(--design-unit) + var(--focus-stroke-width) - var(--stroke-width))* 0.7px) 0;
```

Comment:
If I have a very long line I am assuming this bubble will flow off the visible page height? Is it. Is it possible with the hover mechanics that we have to make it so that we can scroll the wrapped log lines within some sensible limit so that doesn't happen or perhaps make it possible to copy that log line from that location in the clip board?

Reply:
There is a 150 char limit on the displayed log entry tooltip before an ellipsis. If you want more detail then you click on the button.